### ISSUE TITLE
"MAJ.MIN.PATCHsvn" used for LLVM local builds confuses get_llvm_config

### ISSUE THREAD

**Issue by pnkfelix** [gsingh93/llvm#1]
Note: I know this is a bug in `llvm_sys`, not `llvm`.  But, `llvm_sys` is hosted on bitbucket (which has no integrated issue tracker) and no link I could see to an associated issue tracker.

So I'm filing this issue here.

---

Trying to use `llvm` crate with a locally built LLVM resulting in this panic:

```
thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: IncorrectParse(Version { major: 3, minor: 8, patch: 0, pre: [], build: [] }, "3.8.0svn")', ../src/libcore/result.rs:731
```

The issue is that the LLVM build generates a version string of the form "MAJ.MIN.PATCHsvn" (in this case, "3.8.0svn") that is not compatible with what semver is expecting.

I am currently working around this locally by changing my LLVM build to insert a dash before the "svn".  But it may be better to change `get_llvm_config` to, um, "clean up" the input string before handing it over to the semver parser.

**Comment by gsingh93** [comment]
@tari is the maintainer of `llvm_sys`, he should be able to look into this. If he doesn't see this Github ping for some reason, he's also on IRC under the same name.

**Comment by tari** [comment]
Thanks for the ping; I apparently never enabled the issue tracker on bitbucket before, but I have now.

Issue at https://bitbucket.org/tari/llvm-sys.rs/issues/2

**Comment by tari** [comment]
Should be fixed in version 0.1.4 of `llvm_sys`, which I've just published.
